SPM Example/Tests/e2e/ProviderTests.swift

```diff
@@ -9,23 +9,37 @@ import PortalSwift
 @testable import SPM_Example
 import XCTest
 
+@available(iOS 16.0, *)
```

Comment:
How come this is needed?

Reply:
I am planning on adding Passkey tests, but didn't get to it in this pass.